CHANGES.rst

```diff
@@ -228,6 +228,14 @@ API Changes
   - Moved ``units.cgs.emu`` to ``units.deprecated.emu`` due to ambiguous
     definition of "emu". [#4918, #5906]
 
+  - ``jupiterMass``, ``earthMass``, ``jupiterRad``, and ``earthRad`` no longer
+    have their prefixed units included in the standard units.  If needed, they
+    can still  be found in ``units.deprecated``. [#5661]
+    
+  - ``solLum``,``solMass``, and ``solRad`` no longer have  their prefixed units
+    included in the standard units.  If needed, they can still be found in  
+    ``units.vounit_needed``, and are enabled by default. [#5661]
```

Comment:
@eteq , these are not the *only* units in VO standards though. Naming these "required by VO" implies others are not required, unless you want to put *all* VO units here. It's a can 'o worms.